akka-stream/src/main/scala/akka/stream/impl/io/TcpStages.scala

```diff
@@ -273,10 +276,10 @@ private[stream] object TcpConnectionStage {
       override def onUpstreamFailure(ex: Throwable): Unit = {
         if (connection != null) {
           if (interpreter.log.isDebugEnabled) {
-            interpreter.log.debug(
-              "Aborting tcp connection because of upstream failure: {}\n{}",
-              ex.getMessage,
-              ex.getStackTrace.mkString("\n"))
+            val msg = "Aborting tcp connection to {} because of upstream failure: {}"
```

Comment:
Thanks for keeping this, if I remember correctly, it was really frustrating trying to debug why something wasn't working due to the error being swallowed.